src/sound_openal.cpp

```diff
@@ -295,14 +293,6 @@ class OpenALSoundManager: public ISoundManager
 			return;
 		}
 
-		if(alcIsExtensionPresent(m_device, "EXT_vorbis")){
```

Comment:
OpenAL docs: "This function queries if a specified 
context extension is available"
I'm pretty confident.